CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+BUG FIXES:
```

Comment:
Should we go ahead and add the version title so we don't have to open another PR for that?

Reply:
Added next release version